data/json/uncraft/ammo/9mm.json

```diff
@@ -3,28 +3,28 @@
     "result": "9mm",
     "copy-from": "ammo_cartridge",
     "type": "uncraft",
-    "using": [ [ "ammo_9mm", 1 ], [ "ammo_bullet", 1 ] ],
+    "using": [ [ "ammo_9mm", 1 ], [ "ammo_bullet", 2 ] ],
```

Comment:
Please revert. The goal is to eventually balance around the `ammo_bullet` requirement group.